### ISSUE TITLE
How to support accessibility? 

### ISSUE THREAD

**Issue by sonicvision** [kristerkari/react-native-svg-transformer#87]
Hello,

I am using this library, and would like to mention some accessibility text for the svg.
Is there a way to do the same?

**Comment by kristerkari** [comment]
Thanks @sonicvision, that's a good question. 

This library takes regular SVG images and transforms them into a format that React Native can use. As far as I know, React Native has it's own built in accessibility features. That means that if the text inside the SVG image is not readable by iOS' VoiceOver or Android's TalkBack, the you could try to add the `accessibilityLabel` prop to the elements (https://reactnative.dev/docs/accessibility.html), or alternatively use `react-native-svg` directly to get the elements to be accessible (https://github.com/react-native-community/react-native-svg#usage).

**Comment by JiboStore** [comment]
Hi, I'd like to refer to the filename itself from the component to uniquely identify the <SvgComponent/> is there a way to do so?

```
import Logo from 'logo.svg';

<Logo accessibilityLabel={Logo.getFilenameFuncOrSomethingRatherThanHardcodeLogoDotSvg()}
```

**Comment by kristerkari** [comment]
@JiboStore not at least directly using this library. You would need to come up with a custom solution.
